xdsl/transforms/varith_transformations.py

```diff
@@ -15,7 +15,8 @@
 
 # map the arith operation to the right varith op:
 ARITH_TO_VARITH_TYPE_MAP: dict[
-    type[arith.BinaryOperation[Attribute]], type[varith.VarithOp]
+    type[arith.SignlessIntegerBinaryOperation | arith.FloatingPointLikeBinaryOperation],
```

Comment:
it is worth making a type alias for this?

Reply:
IDK, I wouldn't bother